### ISSUE TITLE
create on_plugin_disable signal (Trac #26)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#30]
Migrated from http://microfluidics.utoronto.ca/ticket/26
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "create on_plugin_disable signal", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T03:26:49", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T05:05:05: 
* christian@fobel.net commented:

> commit 3885a8809ac07bf7de3ff961826fdc1ba24ee44a
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Jan 3 23:56:50 2012 -0500
> 
>     Create on_plugin_disable signal...
>     
>     Create on_plugin_disable signal for all plugins.  Combined with the
>     on_plugin_enable signal, this allows for a plugin to initialize when enabled and
>     perform any necessary clean-up when being disabled.  Fixes #30.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
